scipy/linalg/_decomp.py

```diff
@@ -28,10 +28,15 @@
 from .lapack import get_lapack_funcs, _compute_lwork
 from scipy._lib.deprecation import _NoValue, _deprecate_positional_args
 
+from scipy._lib._array_api import array_namespace, is_numpy
 
 _I = numpy.array(1j, dtype='F')
 
 
+def arg_err_msg(param):
```

Comment:
will be renamed and imported from `_array_api.py` in a follow up to https://github.com/scipy/scipy/pull/19005.